### PR TITLE
Update landing page, navbar, supported majors, etc.

### PR DESCRIPTION
**Pull request by churichard**
New landing page:

image

About page: 

image

Privacy Policy:

image

### CODE REVIEW

File: tigerpath/templates/tigerpath/about.html [hunk 1]
```diff
@@ -18,38 +18,57 @@
...
-        <p>We currently fully support Woody Woo, COS AB, and the six BSE majors (CBE, CEE, COS, ELE, MAE, and ORF), but we are continually adding more majors. If you'd like to request for a major to be added to TigerPath, please fill out <a href="https://goo.gl/forms/pKxjmubIOSCOeR8L2" target="_blank">this form</a>!</p>
+        <p>We built TigerPath because we saw a need for Princeton students to have a tool that could help them think long-term about what courses to take and how to fulfill their graduation requirements.</p>
+        <p>Many students try to manually manage what courses they want to take and when they're taking them by using an Excel spreadsheet or pen-and-paper, but any mistake can easily lead to an overloaded semester or a need to take summer classes in order to fulfill missed requirements.</p>
```
Comment (bnehoran) [inline tigerpath/templates/tigerpath/about.html]: No need to use the "they" pronoun
```suggestion
        <p>Many students try to manually manage what courses to take and when to take them by using an Excel spreadsheet or pen-and-paper, but any mistake can easily lead to an overloaded semester or a need to take summer classes in order to fulfill missed requirements.</p>
```

File: tigerpath/templates/tigerpath/about.html [hunk 2]
```diff
@@ -18,38 +18,57 @@
...
+        <p>TigerPath aims to solve this problem by providing a platform for students to manage what courses they're taking during each semester and giving them real-time feedback on the major requirements their schedule fulfills and does not fulfill. Our goal is for TigerPath to become a useful, enduring tool that ensures Princeton students are able to stay on top of requirements and graduate on time.</p>
+      </div>
+
+      <div class="text-section">
+        <h4>Supported Majors</h4>
+        <p>We currently fully support the follow majors:</p>
+        <p class="majors-list"><li>Anthropology</li>
```
Comment (bnehoran) [inline tigerpath/templates/tigerpath/about.html]:
```suggestion
        <p class="majors-list">
          <li>Anthropology</li>
```

**Comment by bnehoran** [comment]
I'm not sure I like this new all blue background.
What's wrong with the path image we have now?

**Comment by churichard** [comment]
The background is actually a blue gradient that fades down, which looks a
little better than full blue. I'm open to alternatives on this, though.

I wanted to get rid of the image because it loads slowly and it doesn't
clearly inform potential users what our app is about when they first visit
our website. The new design presents them with the gif immediately and
gives them a better idea of what TigerPath is like.

**Review by bnehoran — CHANGES_REQUESTED** [review]
The home page looks good! I like the updated layout.

I sort of miss the grey navbar, and I also think it should appear on all the pages in order to be consistent with the main page (the page with the user schedule).
Also, the text on the About and Privacy pages is pretty hard to read, so that should be fixed.

**Review by bnehoran — APPROVED** [review]

**Pull request merged**